google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageReadChannel.java

```diff
@@ -41,27 +42,41 @@
 @ThreadSafe
 final class CloudStorageReadChannel implements SeekableByteChannel {
 
-  private final ReadChannel channel;
+  private ReadChannel channel;
   private long position;
   private long size;
-
+  private Storage gcsStorage;
```

Comment:
@jean-philippe-martin I didn't notice this Codacy recommendation before, could you do this?